### ISSUE TITLE
Installer problem

### ISSUE THREAD

**Issue by wardienet** [watchforstock/evohome-client#7]
Hi Andrew - great concept, just trying to get it loaded into OS X to try out. I got this error on install any ideas? Cheers Paul.
Mac-Pro:Python paul$ python3.4 -m pip install ./evohome-client
Exception:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/basecommand.py", line 232, in main
    status = self.run(options, args)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/commands/install.py", line 305, in run
    name, None, isolated=options.isolated_mode,
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/req/req_install.py", line 181, in from_line
    isolated=isolated)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/req/req_install.py", line 54, in __init__
    req = pkg_resources.Requirement.parse(req)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2873, in parse
    reqs = list(parse_requirements(s))
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2820, in parse_requirements
    "version spec")
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2785, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', './evohome-client', 'at', '/evohome-client')

**Comment by watchforstock** [comment]
Hi Paul,

I'm not a Mac user so may now ask some stupid questions:

- Can you successfully install other packages through pip?
- Does the evohome-client folder exist in the directory you're in?
- Has the checkout worked - can you see a setup.py file inside the folder?

Otherwise, the command you're running looks right and I've not heard of others having trouble installing. 

Andrew

**Comment by wardienet** [comment]
Hi Andrew, thanks for your quick response. I'm away from home with work for a couple of days now but when I get back I will check and let you know. I just had a first quick go at installing it so may well have made a mistake like below. I did get pip to work through. I'm an ex developer but years ago and not too familiar with Python but will pick it up again! Great idea this script, I just upgraded my old original EvoHome and impressed by the new one. Just a shame their own iOS app is rubbish. I also use a Netatmo and that has a great mechanism for logging. 
Thanks,
Paul
